app/src/main/java/ryey/easer/core/data/storage/backend/json/condition/ConditionSerializer.java

```diff
@@ -44,10 +44,10 @@ public String serialize(ConditionStructure data) throws UnableToSerializeExcepti
         }
     }
 
-    private JSONObject serialize_condition(ConditionData condition) throws JSONException {
-        JSONObject json_situation = new JSONObject();
-        json_situation.put(C.SPEC, LocalSkillRegistry.getInstance().condition().findSkill(condition).id());
-        json_situation.put(C.DATA, condition.serialize(PluginDataFormat.JSON));
-        return json_situation;
-    }
+    private static JSONObject serialize_condition(ConditionData condition) throws JSONException {
+		JSONObject json_situation = new JSONObject();
+		json_situation.put(C.SPEC, LocalSkillRegistry.getInstance().condition().findSkill(condition).id());
+		json_situation.put(C.DATA, condition.serialize(PluginDataFormat.JSON));
+		return json_situation;
+	}
```

Comment:
Indent is wrong

Reply:
Thank you for your feedback. Yeah, I already know this issue, it's because the tool works through ASTs, and I leave the default reformatting of the library when i write back the changes into source files.

Reply:
You are right... I misspelt the word.